cython_setuptools/vendor.py

```diff
@@ -1,17 +1,15 @@
-import subprocess
+import argparse
```

Comment:
introduce `isort` config before this commit ? may be no required since the default seems sufficient.